### PR TITLE
[hackathon] Change the size of both component-id and token with 16 bytes

### PR DESCRIPTION
**Pull request by mcd500**
Unifying the size with 16 bytes of both component-id (was 15 bytes) and token (8 bytes).
It is for making the example section more realistic.

### CODE REVIEW

File: draft-ietf-teep-protocol.md [hunk 1]
```diff
@@ -1417,8 +1417,8 @@ COSE is shown.
   02                     # unsigned(2) uint (0..23)
   A5                     # map(5)
     14                   # unsigned(20) uint (0..23)
-    48                   # bytes(8) (8..64)
-      A0A1A2A3A4A5A6A7
+    4F                   # bytes(8) (8..64)
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:1420]:
```suggestion
    4F                   # bytes(16) (8..64)
```

File: draft-ietf-teep-protocol.md [hunk 2]
```diff
@@ -1429,10 +1429,10 @@ COSE is shown.
     82                   # array(2)
       81                 # array(1)
         4F               # bytes(15)
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:1431]:
```suggestion
        4F               # bytes(16)
```

File: draft-ietf-teep-protocol.md [hunk 3]
```diff
@@ -1429,10 +1429,10 @@ COSE is shown.
...
-          0102030405060708090A0B0C0D0E0F
+          000102030405060708090A0B0C0D0E0F
       81                 # array(1)
         4F               # bytes(15)
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:1434]:
```suggestion
        4F               # bytes(16)
```

File: draft-ietf-teep-protocol.md [hunk 4]
```diff
@@ -1466,8 +1466,8 @@ COSE is shown.
   03                     # unsigned(3) uint (0..23)
   A3                     # map(3)
     14                   # unsigned(20) uint (0..23)
-    48                   # bytes(8) (8..64)
-      ABA1A2A3A4A5A6A7
+    4F                   # bytes(8) (8..64)
```
Comment (dthaler) [inline draft-ietf-teep-protocol.md:1469]:
```suggestion
    4F                   # bytes(16) (8..64)
```

**Review by dthaler — CHANGES_REQUESTED** [review]

**Comment by mcd500** [comment]
Thanks for the detail review, I have made the changes correctly.
@dthaler please review

**Review by dthaler — APPROVED** [review]

**Pull request merged**